src/iuse.cpp

```diff
@@ -7966,7 +7966,7 @@ washing_requirements washing_requirements_for_volume( units::volume vol )
 {
     int water = divide_round_up( vol, 125_ml );
     int cleanser = divide_round_up( vol, 1000_ml );
-    int time = 1000 * vol / 250_ml;
+    int time = to_turns<int>( 10_seconds * ( vol / 250_ml ) );
```

Comment:
```suggestion
    int time = to_turns<int>( 10_seconds * vol / 250_ml );
```
These brackets are redundant.

Reply:
Clang builds break without the brackets.
```
clang++  -DRELEASE -DGIT_VERSION -DTILES -DBACKTRACE -ffast-math -Os -Wall -Wextra -fsigned-char -Werror -std=c++11 -MMD -MP -I/usr/include/SDL2 -D_REENTRANT -I/usr/include/SDL2 -D_REENTRANT -I/usr/include/SDL2 -D_REENTRANT -c src/iuse_actor.cpp -o obj/tiles/iuse_actor.o
In file included from src/iuse.cpp:20:
In file included from src/artifact.h:8:
In file included from src/itype.h:12:
src/calendar.h:440:20: error: no matching conversion for functional-style cast from 'quantity<decltype(std::declval<int>() * std::declval<int>()), units::volume_in_milliliter_tag>'
      (aka 'quantity<int, units::volume_in_milliliter_tag>') to 'time_duration'
            return time_duration( lhs.turns_ * rhs );
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/iuse.cpp:7969:42: note: in instantiation of function template specialization 'operator*<units::quantity<int, units::volume_in_milliliter_tag> >' requested here
    int time = to_turns<int>( 10_seconds * vol / 250_ml );
                                         ^
src/calendar.h:300:7: note: candidate constructor (the implicit copy constructor) not viable: no known conversion from 'quantity<decltype(std::declval<int>() * std::declval<int>()),
      units::volume_in_milliliter_tag>' (aka 'quantity<int, units::volume_in_milliliter_tag>') to 'const time_duration' for 1st argument
class time_duration
      ^
src/calendar.h:300:7: note: candidate constructor (the implicit move constructor) not viable: no known conversion from 'quantity<decltype(std::declval<int>() * std::declval<int>()),
      units::volume_in_milliliter_tag>' (aka 'quantity<int, units::volume_in_milliliter_tag>') to 'time_duration' for 1st argument
class time_duration
      ^
src/calendar.h:306:28: note: candidate constructor not viable: no known conversion from 'quantity<decltype(std::declval<int>() * std::declval<int>()), units::volume_in_milliliter_tag>'
      (aka 'quantity<int, units::volume_in_milliliter_tag>') to 'const int' for 1st argument
        explicit constexpr time_duration( const int t ) : turns_( t ) { }
                           ^
src/calendar.h:312:9: note: candidate constructor not viable: no known conversion from 'quantity<decltype(std::declval<int>() * std::declval<int>()), units::volume_in_milliliter_tag>'
      (aka 'quantity<int, units::volume_in_milliliter_tag>') to 'const std::nullptr_t' (aka 'const nullptr_t') for 1st argument
        time_duration( const std::nullptr_t ) : turns_( 0 ) { }
        ^
src/calendar.h:309:9: note: candidate constructor not viable: requires 0 arguments, but 1 was provided
        time_duration() = default;
        ^
src/calendar.h:432:46: error: overload resolution selected deleted operator '/'
            return time_duration( lhs.turns_ / rhs );
                                  ~~~~~~~~~~ ^ ~~~
src/iuse.cpp:7969:48: note: in instantiation of function template specialization 'operator/<units::quantity<int, units::volume_in_milliliter_tag> >' requested here
    int time = to_turns<int>( 10_seconds * vol / 250_ml );
                                               ^
src/units.h:215:13: note: candidate function [with lvt = int, ut = units::volume_in_milliliter_tag, rvt = int, $3 = void] has been explicitly deleted
inline void operator/( lvt, quantity<rvt, ut> ) = delete;
            ^
src/calendar.h:268:25: note: candidate function template not viable: no known conversion from 'const int' to 'const time_duration' for 1st argument
constexpr time_duration operator/( const time_duration &lhs, const T rhs );
                        ^
src/units.h:208:1: note: candidate template ignored: could not match 'quantity<type-parameter-0-0, type-parameter-0-1>' against 'const int'
operator/( const quantity<lvt, ut> &lhs, const rvt &divisor )
^
src/units.h:220:1: note: candidate template ignored: could not match 'quantity<type-parameter-0-0, type-parameter-0-1>' against 'const int'
operator/( const quantity<lvt, ut> &lhs, const quantity<rvt, ut> &rhs )
^
2 errors generated.
make: *** [Makefile:768: obj/tiles/iuse.o] Error 1
make: *** Waiting for unfinished jobs....
```

Reply:
Ok, got it.